ClosedXML/Excel/Ranges/XLRangeBase.cs

```diff
@@ -513,18 +513,21 @@ public virtual Boolean IsEmpty(Boolean includeFormats)
 
         public virtual Boolean IsEntireRow()
         {
-            return RangeAddress.FirstAddress.ColumnNumber == 1
-                   && RangeAddress.LastAddress.ColumnNumber == XLHelper.MaxColumnNumber;
+            return RangeAddress.IsEntireRow();
```

Comment:
Add check `RangeAddress.IsValid`, otherwise the exception will occur (I suspect). Same in the next 2 methods

Reply:
Ok, I'll add the `IsValid` check to the methods in `RangeAddress`